src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -33,6 +33,11 @@
 public class ParserUtil {
 
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
+    private static final String RESPONSE_CONSTRAINTS = "Response for criteria should be indicated as y or n";
+
+    public static boolean isValidYesNoResponse(String response) {
```

Comment:
This method can be private since it is only used within the class.

Reply:
I missed that, thanks for spotting.